seabird.proto

```diff
@@ -166,6 +167,14 @@ message StreamInfoResponse {
 
   map<string, CommandMetadata> commands = 3;
 
-  // TODO: find a way to track IPs related to plugins - maybe deny connections
-  // from an IP other than the registration
+  string ip = 4;
+  uint64 connection_time = 5;
```

Comment:
Is this in seconds? Alternatively, we could just return when the stream started.

Reply:
Sorry, this is meant to be the timestamp that the stream started. I'll change the name.